repair/repair.cc

```diff
@@ -1397,29 +1401,39 @@ static int do_repair_start(seastar::sharded<database>& db, sstring keyspace,
     }
 
     std::vector<sstring> cfs;
+    std::vector<schema_ptr> schemas;
     if (options.column_families.size()) {
         cfs = options.column_families;
         for (auto& cf : cfs) {
             try {
-                db.local().find_column_family(keyspace, cf);
+                schemas.push_back(db.local().find_column_family(keyspace, cf).schema());
             } catch(...) {
                 throw std::runtime_error(format("No column family '{}' in keyspace '{}'", cf, keyspace));
             }
         }
     } else {
-        cfs = list_column_families(db.local(), keyspace);
+        std::tie(cfs, schemas) = list_column_families(db.local(), keyspace);
+    }
+    if (cfs.empty()) return id;
```

Comment:
Use {} with if. 

The return here will skip the tracker::run() below which prints repair id has finished successfully, so

```
if (cfs.empty()) {
     rlogger.info("repair id {} completed successfully", id);
     return id;
}

or

if (cfs.empty()) {
     rlogger.info("repair id {} completed successfully: no tables to repair", id);
     return id;
}
```

Reply:
Good idea. Thanks @asias! Done!